public/javascripts/SVValidate/src/label/LabelContainer.js

```diff
@@ -22,6 +22,11 @@ function LabelContainer() {
      * @param labelMetadata     Label metadata (validationProperties object)
      */
     function push(labelId, labelMetadata) {
+        // If the most recent label is the same as current (meaning it was an undo), remove the undo and use this one.
+        const mostRecentLabel = currentLabels[currentLabels.length - 1];
```

Comment:
I added this so that, when a user hits undo and then re-validates the label before we push data to the back end, we can actually use the "update" part of the "insertOrUpdate" function for the validation. It removes the "undo" validation object and just adds this new validation. That way we don't delete the old one and then add a new one, we just update the old one.